libs/api/Verifier.cpp

```diff
@@ -649,17 +649,17 @@ _ebpf_api_elf_verify_program_from_stream(
 
     try {
         const ebpf_platform_t* platform = &g_ebpf_platform_windows;
-        ebpf_verifier_options_t verifier_options = ebpf_verifier_default_options;
+        ebpf_verifier_options_t verifier_options{};
         verifier_options.assume_assertions = verbosity < EBPF_VERIFICATION_VERBOSITY_VERBOSE;
-        verifier_options.check_termination = true;
+        verifier_options.cfg_opts.check_for_termination = true;
```

Comment:
nit: though we need `check_for_termination` option only when we are actually verifying the program, would it be less error prone (for any future changes), if we used same const verifier options for all purposes (like enumerating programs, disassembly, etc.)?

Reply:
Not sure.  (But seems separate from this PR which keeps the code the same.)